src/components/Collapsible.stories.tsx

```diff
@@ -18,15 +21,15 @@ export const CollapsibleStory: Story<CollapsibleProps> = (args) => (
   </StoryWrapper>
 );
 
-CollapsibleStory.args = {
+CollapsibleStoryWithIconTrigger.args = {
   disabled: false,
   withTrigger: true,
   label: 'Collapsible List of Items',
 };
 
-CollapsibleStory.argTypes = {
+CollapsibleStoryWithIconTrigger.argTypes = {
   triggerIcon: {
-    options: Object.values(IconName),
+    options: [...Object.values(IconName)],
```

Comment:
qq: why does this need a shallow copy?

Reply:
ah whoops, it doesn't - remenants from playing around with old edit; will remove